Gruntfile.js

```diff
@@ -0,0 +1,28 @@
+// Declare dependencies
+/* global module */
+
+"use strict";
+
+module.exports = function (grunt) {
+
+    // Project configuration.
+    grunt.initConfig({
+        // Project package file destination.
+        pkg: grunt.file.readJSON("package.json"),
+        eslint: {
+            all: ["src/**/*.js", "tests/**/*.js", "demos/**/*.js", "examples/**/*.js", "*.js"]
+        },
+        jsonlint: {
+            all: ["package.json", ".jshintrc", "src/**/*.json", "tests/**/*.json", "demos/**/*.json", "!node_modules", "!src/lib/**", "!tests/lib/**"]
```

Comment:
We don't need these entries: `"demos/**/*.json", "!node_modules", "!src/lib/**", "!tests/lib/**"`